src/0_config.js

```diff
@@ -5,5 +5,5 @@ goog.provide('config');
 
 config.app_service_endpoint = 'https://app.link';
 config.link_service_endpoint = 'https://bnc.lt';
-config.api_endpoint = 'https://api2.branch.io';
+config.api_endpoint = 'https://nightlyregression.api.beta.branch.io';
```

Comment:
This needs to remain 'https://api2.branch.io';

Reply:
Monday morning